rskj-core/src/test/java/co/rsk/peg/federation/FederationTestUtils.java

```diff
@@ -255,4 +257,19 @@ private static Script createInputScriptSig(
             .data(fedRedeemScript.getProgram())
             .build();
     }
+
+    public static Script getFederationMembersP2SHScript(ActivationConfig.ForBlock activations, Federation federation) {
```

Comment:
Should we have this here? Not sure the value it adds as a util. It is only used for a specific case, when storing the last retired fed script